tests/src/Kernel/ProjectRenderTest.php

```diff
@@ -172,7 +172,7 @@ public function testProjectTeaser(): void {
           'items' => [
             [
               'label' => 'Project locations',
-              'body' => '1000 Brussels, Belgium',
+              'body' => 'Belgium',
```

Comment:
Try adding more project locations so we make sure the rendering works as expected including the separator, not just that it renders only the country.

Reply:
That is tested in the dedicated test for the fomatter. see AddressCountryInlineFormatterTest
Here we don't have to re-test it an iterate through this big test if you don't mind.

Reply:
Fine.